collections.yaml

```diff
@@ -1,5 +1,6 @@
 collections:
     - title: How To Use Search Patterns on Sourcegraph
+      slug: how-to-use-search-patterns-on-sourcegraph
```

Comment:
I've changed this to be required as I think it could be useful to have down the road.